webpack/assets/javascripts/react_app/components/AutoComplete/components/AutoCompleteSearchButton.js

```diff
@@ -3,22 +3,24 @@ import PropTypes from 'prop-types';
 import { Button, Col, Icon } from 'patternfly-react';
 import { translate as __ } from '../../../common/I18n';
 
-const SearchButton = ({ className, ...props }) => (
+const SearchButton = ({ className, children, ...props }) => (
   <Button {...props} className={`autocomplete-search-btn ${className}`}>
     <Icon name="search" />
     <Col className="autocomplete-search-btn-text" xsHidden>
       &nbsp;
-      {__('Search')}
+      {children}
     </Col>
   </Button>
 );
 
 SearchButton.propTypes = {
   className: PropTypes.string,
+  children: PropTypes.node,
 };
 
 SearchButton.defaultProps = {
   className: '',
+  children: __('Search'),
```

Comment:
Basically, this is what this PR allows us to do :)

Reply:
how does moving the mocks from `test_helpers` impacts on PropTypes?

Reply:
I didn't really get it but as far as it weird, it seems the `test_setup.js` running after the component file gets loaded.